scripts/update-generator/main.go

```diff
@@ -175,7 +123,7 @@ func run() error {
 
 	installerPath := flag.Arg(0)
 
-	binaries := flag.Args()[1:]
+	target := flag.Args()[0]
```

Comment:
I think the `installerPath` is not used anymore, so it can be removed here, and also in the `createUpdate` function.